plugins/MantisCoreFormatting/MantisCoreFormatting.php

```diff
@@ -123,11 +123,6 @@ private function processMarkdown( $p_string ){
 
 		$t_string = MantisMarkdown::convert_text( $t_string );
```

Comment:
Should we completely remove processMarkdown() and call directly MantisMarkdown::convert_text() and have it do all the processing, e.g. the str_replace just above this line?  This makes the code easier to unit test and remove this unnecessary indirection step.